src/tests/data/test_keys.cairo

```diff
@@ -3,8 +3,8 @@ use starknet::{ContractAddress, contract_address_const};
 
 #[test]
 fn given_constant_keys_when_tested_then_expected_results() {
-    let wnt = keys::wnt();
-    assert(wnt == 0x74d98a44e50c39f5a55e16dec65fc43c09ca04efa146e762ab2e4cf8a02ae0e, 'wrong_key');
+    let fee_token = keys::fee_token();
+    assert(fee_token == 0x74d98a44e50c39f5a55e16dec65fc43c09ca04efa146e762ab2e4cf8a02ae0e, 'wrong_key'); //change?
```

Comment:
what value should this be?

Reply:
Let's put eth address which is `0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7`

Reply:
Oh no sorry it's only returning the key to access the fee token not the address

Reply:
It's just the poseidon hash of FEE_TOKEN, don't really know what it is, you can print `fee_token_key` or panic with its value to see what it is and replace it in the equality

Reply:
This should work `0x3d260a67237e2a5babde11feb7ae50a519c48b36fccb6337e2975e0545410a5` @dbejarano820